README.md

```diff
@@ -57,6 +57,10 @@ Prerequisites:
 
 If you are actively testing code against the Realm source, see also the unit test projects and other tests under the Tests folder.
 
+## Examples
+
+Some minimal examples of Realm use can be found in the `examples` folder.
```

Comment:
You could maybe specifically list them (might not make sense now because there is just one but there will be more in the future) and say what they are about.

Since I am working on a Unity example at the moment there might be quite some differences in those examples.

Which platform are they for?
Which Realm version were they built with (especially if we keep it fixes this could be important to point out early)?
What use cases can be found in this example? Just a simple local Realm, notifications, Sync, etc.
Are there any requirements to use this example? Xamarin v42.3, .NET 23.6
Maybe a summary of what this example is?

edit:
Just saw the summary already being part of the example itself.

I guess the question is: Do we want to keep everything in here in one place to give an overview or is it better placed in each example to only provide information where it is needed?

Reply:
Yes, that's the point. I think maybe just a one liner for each example in the main readme will help, so that they don't need to open each example.

Reply:
Looks good. :)